app.js

```diff
@@ -28,3 +28,93 @@ hamburgerCloseConst.addEventListener('click', closeMenu);
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby On Rails'],
+    liveLink: 'https://examplelinkofpage.com/live',
+    sourceLink: 'https://examplelinkofpage.com/source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    description: ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby On Rails'],
+    liveLink: 'https://examplelinkofpage.com/live',
+    sourceLink: 'https://examplelinkofpage.com/source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    description: ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby On Rails'],
+    liveLink: 'https://examplelinkofpage.com/live',
+    sourceLink: 'https://examplelinkofpage.com/source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    description: ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby On Rails'],
+    liveLink: 'https://examplelinkofpage.com/live',
+    sourceLink: 'https://examplelinkofpage.com/source',
+  },
+  {
+    name: 'Professional Art Printing Data',
+    description: ' A daily selection of privately personalized reads  Lorem Ipsum is simply dummy text of the printing and typesetting industry ',
+    image: 'screen.png',
+    technologies: ['HTML', 'Bootstrap', 'Ruby On Rails'],
+    liveLink: 'https://examplelinkofpage.com/live',
+    sourceLink: 'https://examplelinkofpage.com/source',
+  },
+];
+
+function openPopup(index) {
+  const project = projects[index];
+
+  const popup = document.getElementById('popup');
+  const popupName = document.getElementById('popupName');
+  const popupTechnologies = document.getElementById('popupTechnologies');
+  const popupImage = document.getElementById('popupImage');
+  const popupDescription = document.getElementById('popupDescription');
+  const popupLiveLink = document.getElementById('popupLiveLink');
+  const popupSourceLink = document.getElementById('popupSourceLink');
+
+  const multiPostPage = document.getElementById('multiPostPage');
+
+  popupName.textContent = project.name;
+
+  popupTechnologies.innerHTML = project.technologies.map((tech) => `<li>${tech}</li>`).join('');
+  popupImage.src = project.image;
+
+  popupDescription.textContent = project.description;
+  popupLiveLink.onclick = () => window.open(project.liveLink);
+  popupSourceLink.onclick = () => window.open(project.sourceLink);
+
+  popup.classList.add('open-popup');
+  multiPostPage.classList.add('popup-container');
+}
```

Comment:
Your design
![image](https://github.com/bhushan354/Portfolio_Project/assets/78251927/ff6e38a3-45d9-42bf-9d8f-45ee85430e54)

Figma
![image](https://github.com/bhushan354/Portfolio_Project/assets/78251927/23b8da09-5a6a-40b1-be37-0ace3644952f)

Please, let's stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Layout
- Spacing (margin & padding)
- Content sizes (Both Mobile and Desktop)


The issue with the popup window 👀

![image](https://github.com/bhushan354/Portfolio_Project/assets/78251927/c501c7c4-f581-46d8-8ae6-9e0bd559fdb5)

- I noticed that only the first project card works, the popup window does not display for the remaining cards, whereas our objective is to associate each project with the popup window details.

- Please kindly fix this issue 💪